mapdata/WarcraftLegacies/UpgradeData/Skin/1735222610.json

```
@@ -19,6 +19,48 @@
       "Id": 829122663,
       "Type": 3,
       "Value": "A"
+    },
+    {
+      "Level": 2,
+      "Id": 829579623,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNUpgradeMoonGlaive.blp"
+    },
+    {
+      "Level": 2,
+      "Id": 1718840935,
+      "Type": 3,
+      "Value": "(Light Blue Research)"
+    },
+    {
+      "Level": 2,
+      "Id": 829122663,
+      "Type": 3,
+      "Value": "A"
+    },
+    {
+      "Level": 2,
+      "Id": 1835101799,
+      "Type": 3,
+      "Value": "Upgrade Moon Glaive"
+    },
+    {
+      "Level": 2,
+      "Id": 829453415,
+      "Type": 3,
+      "Value": "Upgrade Moon Glaive [|cffffcc00Level 2|r]"
+    },
+    {
+      "Level": 2,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Causes Huntress attacks to strike an additional unit"
```

Huntress should be in the possessive, plural form. Upgrade descriptions are full sentences so should end with a full stop. Thus: 
```suggestion
      "Value": "Causes Huntresses' attacks to strike an additional unit."
```